evap/evaluation/tools.py

```diff
@@ -11,6 +11,18 @@
 from django.utils.translation import LANGUAGE_SESSION_KEY, get_language
 
 
+def is_prefetched(instance, attribute_name):
+    """
+    Is the given attribute prefetched? Can be used to do ordering or counting
+    in python and avoid additional database queries
+    """
+
+    if hasattr(instance, '_prefetched_objects_cache') and attribute_name in instance._prefetched_objects_cache:
```

Comment:
`return hasattr...`

this method might have been useful elsewhere, why didn't you add it earlier?!